docs/integrations/wallets.md

```diff
@@ -0,0 +1,108 @@
+# Wallets
+
+If you are an Ethereum wallet provider, you can integrate Lido staking into your app or website and earn [Lido referral rewards](https://lido.fi/referral). 
+
+## Your referral link
+
+You can start referring people simply by sharing your special referral link to Lido [Staking Widget](https://stake.lido.fi/). Your referral link looks like this `https://stake.lido.fi/?ref=YOUR_REWARDS_ADDRESS` where `YOUR_REWARDS_ADDRESS` is your Ethereum address or ENS name registered for the Lido Referral Program. Anyone who uses your link to stake will automatically set your rewards address as their referral.
```

Comment:
Should note that need to apply for referral program explicitly since it has a whitelisting policy :eye: